File: examples/petstore-sdk/go/sdk/pet.go
package petstore

type Pet struct {
	Id *int64 `json:"id,omitempty"`
	Name string `json:"name"`
	Category *Category `json:"category,omitempty"`
	PhotoUrls []string `json:"photoUrls"`
	Tags []Tag `json:"tags,omitempty"`
	Status *PetStatus `json:"status,omitempty"`
}

File: examples/petstore-sdk/kotlin/sdk/com/example/petstore/model/Pet.kt
package com.example.petstore.model

import kotlinx.serialization.Serializable

@Serializable
data class Pet(
    val id: Long?,
    val name: String,
    val category: Category?,
    val photoUrls: List<String>,
    val tags: List<Tag>?,
    val status: Pet_Status?,
)
